utils/dates-to-epiweek.R

```diff
@@ -24,7 +24,7 @@
 #   dplyr::summarise(weekly_incidence = sum(daily_incidence))
 dates_to_epiweek <- function(df){
   
-  seq <- tibble::tibble(date = unique(df$date),
+  seq <- tibble::tibble(date = as.Date(unique(df$date)),
```

Comment:
@nikosbosse thanks for updating this this week! this seemed to be causing a bug, so just flagging as i guess you may need to check this in the DE/PL forecast code.

Reply:
Nice spot.